script/pre-commit

```diff
@@ -33,7 +30,7 @@ then
 	fi
 	
 	# Run rubocop on the staged files
-	bin/bundle exec rubocop ${FILES}
+	rubocop ${FILES}
```

Comment:
I kinda made this change locally, but is this what Eugene suggested?

This won't work on staged files (as we've discussed), and this changes behaviour to only the ones we include.

Reply:
will do some more testing with this. We ran rubocop by calling `ruby rubocop.rb` before in flipp project.